lib/Email/MIME.pm

```diff
@@ -542,6 +556,34 @@ sub content_type_attribute_set {
   $self->_compose_content_type($ct_header);
 }
 
+=method encode_check
+
+=method encode_check-set
+
+  $email->encode_check;
+  $email->encode_check_set(0);
+  $email->encode_check_set(Encode::FB_DEFAULT);
+
+Gets/sets the current C<encode_check> setting (default: I<FB_CROAK>).
+This is the parameter passed to L<Encode/"decode"> an L<Encode/"encode">
+when C<body_str()>, C<body_str_set()>, and C<create()> are called.
+
+See L<Encode> for more information.
```

Comment:
Better link would be directly to Handling Malformed Data section...
I hope this `L<Encode|Encode/Handling Malformed Data>` could work.